mne/decoding/transformer.py

```diff
@@ -248,6 +248,12 @@ def inverse_transform(self, X, y=None):
 class PSDEstimator(TransformerMixin):
     """Compute power spectrum density (PSD) using a multi-taper method
 
+    This structures data so that it can be easily incorporated into
+    scikit-learn pipelines. It relies heavily on the multitaper_psd
```

Comment:
hmm ... actually, thinking about it some more -- the problem with a `method` argument is that you will have to pass the different options for `multitaper` / `welch` too. Maybe, we can leave it for now ...